src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int MAX_ITEMS = 100;
+    private static final String TOP_DIVIDER = "════════════════════════════════════════════════════════════\n";
+    private static final String BOTTOM_DIVIDER = "\n════════════════════════════════════════════════════════════\n\n";
+    private static final String ADD_TASK_ITEM_DESC = " Roger. I've added this task:\n  ";
+    private static final String LIST_TASK_ITEM_DESC = " Here are the tasks in your list:\n";
+    private static final String COMMAND_MARK_TASK = "mark";
+    private static final String COMMAND_UNMARK_TASK = "unmark";
+    private static final String MARK_TASK_DESC = " Good Job! I've marked this task as completed:\n  ";
+    private static final String UNMARK_TASK_DESC = " Noted, I have marked this task as incomplete:\n  ";
+    private static final String WHITESPACE = " ";
+    private static final String DOT = ".";
+    private static final String LIST_TASK_ITEM_BOTTOM_DIVIDER
+            = "════════════════════════════════════════════════════════════\n\n";
+    private static final String WELCOME_MESSAGE = " Hello! I'm Chatty\n How can I help you?";
+    private static final String COMMAND_LIST_TASK_ITEM = "list";
+    private static final String COMMAND_EXIT = "bye";
+    private static final String EXIT_DESC = " Goodbye. Hope to see you again soon!";
+    private static final String CUR_NO_OF_TASK_START_DESC = "\n You currently have ";
+    private static final String CUR_NO_OF_TASK_END_DESC = " task(s) in the list.";
+    private static final String COMMAND_ADD_TODO = "todo";
+    private static final String COMMAND_ADD_DEADLINE = "deadline";
+    private static final String COMMAND_ADD_EVENT = "event";
```

Comment:
You can consider reducing the number of defined constants to improve readability, and you can consider writing `DESC` as `DESCRIPTION` instead to make it clearer, it is good that your constant names must be all uppercase using underscore to separate words.